docs/section19/19-10.md

```diff
@@ -0,0 +1,84 @@
+        This segment creates a list of baseline hvac system IDs in which G3.1.2.6 exception #2 is likely applicable, the exceptions states "where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems. This exception shall only be used if the system in the proposed design does not use an economizer. If the exception is used, an economizer shall not be included in the baseline building design."
+        - For each building in the B_RMI: `for bldg in B_RMI...Building:`  
+            - Check of the building has refrigerated cases: `if bldg.refrigerated_cases != Null:`  
+                - For each building segment in the building: `for bldg_seg in bldg.building_segments:`  
+                    - Check if the building area type is retail: `if bldg_seg.lighting_building_area_type == "RETAIL":`  
+                        - Create dictionary with hvac iD and zone id list: `hvac_zone_list_w_area_dict = get_hvac_zone_list_w_area(B_RMI)` 
+                        - For each hvac system in the building segment: `for hvac_b in bldg_seg.heating_ventilating_air_conditioning_systems:`  
+                            - Reset proposed_has_economizer boolean variable to FALSE: `proposed_has_economizer =  FALSE`  
+                            - Get zone ids associated with the HVAC system: `zone_list_b = hvac_zone_list_w_area_dict[hvac_b.id]["ZONE_LIST"]`  
+                            - Check if any of the zones are served by an economizer in the proposed design model, start by cycling through each zone served by the baseline HVAC system: `for zone_b in zone_list_b:`  
+                                - Get analogous zone ID in the P_RMI: `zone_p = match_data_element(P_RMI,Zone,zone_b)`  
+                                - Get list of HVAC system IDs associated with the zone in the proposed: `hvac_sys_list_p = Get_list_hvac_systems_associated_with_zone(P_RMI, zone_p)`  
+                                - Loop through each HVAC system and check if it is equipped with an economizer, if it is then set proposed_has_economizer equal to TRUE : `for hvac_p in hvac_sys_list_p:`  
+                                    - Convert hvac_p (which is an ID) to an HVAC object (however the RCT team does this, if this is even needed): `hvac_p_obj = hvac_p`  
+                                    - Check if the HVAC system is equipped with an economizer: `if hvac_p_obj.fan_system.AirEconomizer != Null AND hvac_p_obj.fan_system.AirEconomizer.type != "FIXED_FRACTION": proposed_has_economizer = TRUE`  
+                            - Check if the proposed_has_economizer boolean variable still equals FALSE, if it does then add hvac_b to list of hvac systems where exception #2 is likely applicable: `if proposed_has_economizer == FALSE: hvac_system_exception_2_list.append(hvac_b)`
```

Comment:
As a side note, this logic means that if at least one zone served by the baseline HVAC system has an economizer if the proposed design, the baseline system serving this zone will also be expected to have an economizer even if other zones that it serves do not. I think it's fair, but the current language in App G is ambiguous on that. This will be addressed by the addendum that I drafted, but perhaps we should do an interpretation request for that.